websites/K/Kahoot/presence.ts

```diff
@@ -38,153 +37,378 @@ async function getStrings() {
 			ndString: "kahoot.ndString",
 			rdString: "kahoot.rdString",
 			topX: "kahoot.topX",
+			onPodium: "kahoot.onPodium",
 			of: "kahoot.of",
-			questionNumber: "kahoot.questionNumber"
+			questionNumber: "kahoot.questionNumber",
+			feedback: "kahoot.feedback",
+			waitingAnswer: "kahoot.waitingAnswer",
+			drumRoll: "kahoot.drumRoll",
+			position: "kahoot.position",
+			teamTalk: "kahoot.teamTalk",
+			gameSummary: "kahoot.gameSummary",
+			login: "kahoot.login",
+			createHome: "kahoot.createHome",
+			discover: "kahoot.discover",
+			searchKahoots: "kahoot.searchKahoots",
+			kahootDetails: "kahoot.kahootDetails",
+			kahootProfile: "kahoot.kahootProfile",
+			myKahoots: "kahoot.myKahoots",
+			userReports: "kahoot.userReports",
+			myCourses: "kahoot.myCourses",
+			editingCourse: "kahoot.editingCourse",
+			viewingCourse: "kahoot.viewingCourse",
+			editingKahoot: "kahoot.editingKahoot",
+			previewingKahoot: "kahoot.previewingKahoot",
+			liveCourse: "kahoot.liveCourse",
+			liveCourseActivity: "kahoot.liveCourseActivity"
 		},
 		await presence.getSetting<string>("lang")
 	);
 }
 
-let currentQuestion: string,
-	score: string,
-	ranking: string,
-	top5: Promise<boolean>,
-	strings: { [key: string]: string },
+let strings: { [key: string]: string },
```

Comment:
this type is wrong, it should be more specific

Reply:
This was from the original codebase. What do you suggest I change it to, or how can I make it more specific?

Reply:
Follow the standard in other popular presences by using ReturnType